brandingbrand/flagship
# fix(fssalesforce): fix cart metadata being lost in update cart calls

The Commerce Cloud cart normalizer expects product_details to be set on the cart object. product_details is the result of the operation to request additional product information for the items in the cart. Due to an order of operations problem, a couple cart operations set this data after the normalizer was called, rendering it useless.

Test:
1. Add multiple items to cart
2. Increase the quantity of one of the items in cart
3. Verify that items still have images
4. Remove one of the items
5. Verify that the remaining items still have images
